src/android/barcodescanner.gradle

```diff
@@ -8,7 +8,7 @@ repositories{
 }
 
 dependencies {
-    compile 'com.android.support:support-v4:26.+'
+    compile 'com.android.support:support-v4:26.0.+'
```

Comment:
I think is not required the `.+` since the full version is already defined. It is used to get the upper version in this point.

Reply:
I think Camila is correct

Reply:
It's not a full version, patch version is not specified (it can be: 26.0.0, 26.0.1, 26.0.2).

Reply:
@matskiv it cannot be 4:26.1 because you put 4:26.0.+, it could be this values in the way that it was before 4:26.+ where the `+` will be replaced for the upper value

Reply:
@camilamacedo86 notice that it is a semver versioning after "support-v4:" - major.minor.patch. And I left patch unspecified and put + there.

Reply:
btw: you can find the available version here: https://dl.google.com/dl/android/maven2/index.html

Reply:
@matskiv catcher !! the v4 is out.